src/vulnpy/trigger/xpath.py

```diff
@@ -0,0 +1,45 @@
+import lxml.etree
+import xml.etree.ElementTree as ET
+from io import StringIO
+
+xml = (
+    "<planets>"
+    "<planet>\
+    <name>Kepler</name>\
+</planet>\
+<planet>\
+    <name>Kepler</name>\
+</planet>\
+<planet>\
+    <name>Newton</name>\
+</planet>"
+    "</planets>"
+)
+
+
+def do_lxml_etree_findall(user_input):
+
+    tree = lxml.etree.parse(StringIO(xml))
+
+    tree.xpath('/planet[name[text()="Kepler"]]')
```

Comment:
so admittedly I'm not too familiar with xml parsing or this vulnerability type. Would it make sense to move the initial parsing and call to `xpath` to the module level instead of inside of each function?